### DISCUSSION TITLE
OpenBriefcaseFailed

### DISCUSSION THREAD

**Discussion by VWlf** [iTwin/itwinjs-core#6774] [Q&A]
When I attempt to upload a model with a size of approximately 500MB, an error occurs during the process, with the following error message:
![微信图片_20240530105009](https://github.com/iTwin/itwinjs-core/assets/49948146/aadf7047-45b5-4035-8e61-1e5cb6f2c3bc)
It's peculiar that I am able to successfully upload smaller models, but encounter issues with larger ones. Are there any possible reasons for this? `@banked` is using version V2.12.0
![微信图片_20240530105717](https://github.com/iTwin/itwinjs-core/assets/49948146/0af31027-51be-4a49-b8a4-53c74ac2cc9c)

![微信图片_20240530105815](https://github.com/iTwin/itwinjs-core/assets/49948146/e7019cf1-ccfd-4ccf-a140-bdc0397cce52)

**Comment by aruniverse** [comment]
iTwin.js 2.x reached end of life yrs ago, and we are not continuing to support it. Can you try updating to the latest to see if the issue still perists?
